### DISCUSSION TITLE
Screwed up PROFILE

### DISCUSSION THREAD

**Discussion by KC5VDJ** [kej715/DtCyber#157] [General]
How do I get rid of an existing PROFILC in nos 2.8.7?

Somehow I screwed it up a while back, and get a database error trying to reconstruct, or really trying to do anything.

My understanding is that it has to be removed from fast attach status, and I'm thinking ISF might be able to init a new one??

**Comment by kej715** [comment]
ISF does not initialize any files. It enters them or removes them from fast
attach status. It will also launch a system startup job and/or startup
procedure at deadstart time. See chapter 20 of the NOS 2 Analysis Handbook
for details about ISF. The NOS 2 Administration Handbook provides
documentation about PROFILE and the PROFILa files. The PROFILE command
initializes PROFILa files.

To remove a file from fast attach status, you must use the
IDLEFAMILY command to idle the family on which the file resides and them
use X.ISF(R=filenam) to remove the file from fast attach status. This is
described in chapter 20 of the NOS 2 Analysis Handbook.

**Comment by KC5VDJ** [comment]
Thanks.  Been stuck but back burner on this for a while.
